hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -847,6 +856,16 @@
       stage: V3
       tests: []
     }
+    {
+      name: chip_sw_plic_alerts
+      desc: '''Verify alerts from PLIC due to both, TL intg and reg WE onehot check faults.
```

Comment:
The all_escalation_reset test should do reg WE onehot: https://github.com/lowRISC/opentitan/issues/15649.

Reply:
Aah sounds good. When it gets updated to hit the one in RV_PLIC, we can map it to this testpoint.